app/assets/stylesheets/colors.scss

```diff
@@ -0,0 +1,11 @@
+// Text Colors
+$text: #292B31;
```

Comment:
Thanks for this! When I was using a color picker to grab samples from your mockup, I found myself reusing a color a couple times and wondered if I should put it into a variable. 👍

I'm glad to have the actual colors here, instead of whatever approximation I got via a color picker from an image.